### PR TITLE
add training resume, best model save, vdl, remained time computation

### PR DESCRIPTION
**Pull request by wuyefeilin**

### CODE REVIEW

File: dygraph/utils/utils.py [hunk 1]
```diff
@@ -78,6 +78,24 @@ def load_pretrained_model(model, pretrained_model):
                     pretrained_model))
 
 
+def resume(model, optimizer, resume_model):
+    if resume_model is not None:
```
Comment (nepeplwu) [inline dygraph/utils/utils.py:84]: else的逻辑需要处理

File: dygraph/train.py [hunk 2]
```diff
@@ -104,6 +112,17 @@ def parse_args():
...
+    parser.add_argument(
+        '--log_steps',
+        dest='log_steps',
+        help='Display logging information at every log_steps',
+        default=10,
+        type=int)
+    parser.add_argument(
+        '--use_vdl',
+        dest='use_vdl',
+        help='Whether to record the data during training to VisualDL',
```
Comment (nepeplwu) [inline dygraph/train.py]: Whether to record the data to VisualDL during training

File: dygraph/utils/utils.py (hunk 1 above)
Comment (wuyefeilin) [reply dygraph/utils/utils.py:84]: 已处理

File: dygraph/train.py (hunk 2 above)
Comment (wuyefeilin) [reply dygraph/train.py]: 已更改

**Pull request merged**